Lib/unittest/case.py

```diff
@@ -882,24 +882,32 @@ def assertNotEqual(self, first, second, msg=None):
             raise self.failureException(msg)
 
     def assertAlmostEqual(self, first, second, places=None, msg=None,
-                          delta=None):
+                          delta=None, rel_delta=None):
         """Fail if the two objects are unequal as determined by their
-           difference rounded to the given number of decimal places
-           (default 7) and comparing to zero, or by comparing that the
-           difference between the two objects is more than the given
-           delta.
+           difference by one of the following methods:
 
-           Note that decimal places (from zero) are usually not the same
-           as significant digits (measured from the most significant digit).
+           1) Given number of decimal places. This is the default method
+              (default value 7 digits). Note that decimal places (from
+              zero) are usually not the same as significant digits
+              (measured from the most significant digit).
+
+           2) Comparing that the absolute difference between the two
+              objects is more than the given delta.
+
+           3) Comparing that the relative difference between the two
+              objects is more than the given rel_delta. The relative
+              difference is applied with respect to the first object,
```

Comment:
Is there any good reason to apply the difference with respect only to first object?
math.isclose() is symmetrical and I think having different behavior here would be potentially confusing.